rulestest.sh

```diff
@@ -58,6 +63,12 @@ echo "Restarting $TARGET_CONTAINER"
 podman run \
     --name $TARGET_CONTAINER \
     --pod cryostat \
-    --env JMX_PORT=9093 \
-    --env USE_AUTH=true \
-    --rm -d quay.io/andrewazores/vertx-fib-demo:0.6.0
+    --env HTTP_PORT=8081 \
+    --env JMX_PORT=9094 \
+    --rm -d quay.io/andrewazores/vertx-fib-demo:0.7.0
+
+sleep 5
+echo "Deleting rule definition"
```

Comment:
Now that I think about it, maybe this shouldn't delete the rule definition? Since then the periodic archiver doesn't get to run, which I think is probably more important than excersising the deletion function. Alhough, maybe to test both, the `archivalPeriodSeconds` for the rule could just be made to something small like 5 seconds, and then there could be a `sleep 30` before the deletion call? I usually change it to 5 seconds anyway to see the archiver working, though I suppose it is a little unrealistic.

Reply:
Sure - `rulestest.sh` is just meant as a temporary testing aid while the autorules branch is in progress, and it'll be removed before autorules is merged into main. Whatever behaviours and workflows make sense to test will/should be included in AutoRulesIT instead. I'll open a small PR to restore rulestest.sh back to what it was before this PR.